src/components/Input/index.js

```diff
@@ -60,6 +62,9 @@ export default class Input extends PureComponent {
         {error &&
           <span className='input-field__error'>{error}</span>
         }
+        {warning &&
+          <span className='input-field__error'>{warning}</span>
```

Comment:
maybe change the class name to `input-field__warning`?

Reply:
So the developers are able to override the styles outside the library with different classes

Reply:
You're right!